__tests__/integration-tests.test.js

```diff
@@ -138,6 +138,93 @@ describe('GET /api/articles/:article_id tests', () => {
+        .send(newVoteObj)
+        .expect(200)
+        .then((response) => {
+            expect(response.body).toMatchObject(
+                {
+                    article: {
+                        article_id: 1,
+                        title: 'Living in the shadow of a great man',
+                        topic: 'mitch',
+                        author: 'butter_bridge',
+                        body: 'I find this existence challenging',
+                        created_at: '2020-07-09T20:11:00.000Z',
+                        votes: 999,
+                        article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700' 
+                }
+            })
+        })
+    });
+    test('should respond with 200 and an object with the correct information when passed an object with additional propeties', () => {
+        const newVoteObj = {
+             inc_votes : 899,
+             date: 210393,
+             location: "manchester" 
+        }
+        return request(app)
+        .patch("/api/articles/1")
+        .send(newVoteObj)
+        .expect(200)
+        .then((response) => {
+            expect(response.body).toMatchObject(
+                {
+                    article: {
+                        article_id: 1,
+                        title: 'Living in the shadow of a great man',
+                        topic: 'mitch',
+                        author: 'butter_bridge',
+                        body: 'I find this existence challenging',
+                        created_at: '2020-07-09T20:11:00.000Z',
+                        votes: 999,
+                        article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700' 
+                }
+            })
+        })
+    });
+    test('should respond 404 and the message "article does not exist" and when given a valid article_id that does not exist ', () => {
+        const newVoteObj = {
+            inc_votes : 899 
+       }
+       return request(app)
+        .patch("/api/articles/999")
+        .send(newVoteObj)
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("article does not exist")
+        })
+    });
+    test('should respond with a 400 when given a invalid article id', () => {
+        const newVoteObj = {
+            inc_votes : 899 
+       }
+        return request(app)
+        .patch("/api/articles/not-an-id")
+        .send(newVoteObj)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request")
+        })
+    });
+    test('should respond with a 400 when passed an empty object', () => {
+        const newVoteObj = {}
+        return request(app)
+        .patch("/api/articles/1")
+        .send(newVoteObj)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request")
+        })
+    });
+})
+
```

Comment:
Great testing so far, but there is one extra test that can and should be added for this endpoint. 

What should happen when a user makes a request with the request body is missing/ has an incorrect inc_votes property?